userbot/__init__.py

```diff
@@ -186,3 +186,4 @@ def str_to_bool(strbool: str) -> bool:
 MODULE_DESC = {}  # {Module name: MODULE_DESC}
 MODULE_DICT = {}  # {Module name: MODULE_USAGE}
 MODULE_INFO = {}  # {Module name: MODULE_INFO}
+MODULE_DEPS = {}  # {Module name: MODULE_DEPS}
```

Comment:
We don't recommend to use global dicts as any module will be able to overwrite existing keys. Current global dicts in HyperUBot will be removed in future updates

Reply:
I'll probably close the PR now and reopen it once 4.0 is out